node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/NodeResourceLimits.java

```diff
@@ -88,10 +88,13 @@ private long reservedDiskSpaceGb(NodeResources.StorageType storageType, boolean
             return 4;
     }
 
-    private double minRealVcpu(ApplicationId applicationId, ClusterSpec cluster) { return minAdvertisedVcpu(applicationId, cluster); }
+    private double minRealVcpu(ApplicationId applicationId, ClusterSpec cluster) {
+        return minAdvertisedVcpu(applicationId, cluster);
+    }
 
     private double minRealMemoryGb(ClusterSpec cluster) {
-        return minAdvertisedMemoryGb(cluster) - 1.7;
+        if (cluster.type() == ClusterSpec.Type.admin) return 0.95; // TODO: Increase to 1.05 after March 2023
+        return 2.3;
```

Comment:
Some explanation on how these numbers have been found would be nice

Reply:
I added an explanation where they are specified, in CapacityPolicies. The numbers here just lead to a provision failure if they are not met, so they matter less.